templates/zerver/integrations/nagios.md

```diff
@@ -0,0 +1,50 @@
+First, create the stream you’d like to use for Nagios notifications, and subscribe all interested parties to this
+stream. The integration will use the default stream `nagios` if no stream is supplied in the configuration; you
+still need to create the stream even if you are using this default.
+
+Next, download and install our [Python bindings and example scripts](/api) on your Nagios server.
+
```

Comment:
I think my recommendations to use `create-stream.md`, `congrats.md` and to create and use a macro for the instruction to download and install our bindings apply everywhere. :)